CHANGELOG.md

```diff
@@ -4,7 +4,9 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/) starting from 1.x releases.
 
 ### Merged, but not yet released
-> All recent changes are published
+> ~~All recent changes are published~~
+> #### Added
+> - **BREAKING:** Bit depth can now be specified when using `ImageOption` to request PNGs from the Images API.
```

Comment:
Should this be specifically mentioning `8-bit`? Otherwise it sounds like you can now set arbitrarary bit depth's.

Reply:
Good call. will change.